Readme: Update Running on Wayland guide (#4199)
balena-io/etcher: docs/USER-DOCUMENTATION.md

Running on Wayland
------------------

> You can run Etcher on Wayland by passing command line flags

Reviewer: ```suggestion
You can run Etcher on Wayland by passing command line flags like so:
```

```sh

> $ balena-etcher --enable-features=WaylandWindowDecorations --ozone-platform=wayland

Reviewer: ```suggestion
balena-etcher --enable-features=WaylandWindowDecorations --ozone-platform=wayland
```

`$`s make commands harder to copy.

```

> To make this changes permanent to the desktop entry

Reviewer: ```suggestion
To make these changes permanent to the desktop entry, run these commands in your terminal:
```

```sh

> $ cp /usr/share/applications/balena-etcher.desktop ~/.local/share/applications
> $ sed -i 's|^Exec=/opt/balenaEtcher/balena-etcher %U$|Exec=/opt/balenaEtcher/balena-etcher --enable-features=WaylandWindowDecorations --ozone-platform=wayland %U|' ~/.local/share/applications/balena-etcher.desktop

Reviewer: ```suggestion
cp /usr/share/applications/balena-etcher.desktop ~/.local/share/applications
sed -i 's|^Exec=/opt/balenaEtcher/balena-etcher %U$|Exec=/opt/balenaEtcher/balena-etcher --enable-features=WaylandWindowDecorations --ozone-platform=wayland %U|' ~/.local/share/applications/balena-etcher.desktop
```
